api/README.md

```diff
@@ -17,4 +17,4 @@ The feature requires a custom config file named `newspack-popups-config.php` at
 
 ## Disabling the API calls
 
-In high-traffic periods or for debugging purposes, it's possible to disable any calls to the API with the "non-interactive mode" setting (`newspack_popups_non_interative_mode` option).
+In high-traffic periods or for debugging purposes, it's possible to disable any calls to the API with the "non-interactive mode" setting (`newspack_popups_non_interactive_mode` option).
```

Comment:
@jeffersonrabb actually [there's a typo in the option name](https://github.com/Automattic/newspack-popups/blob/1591eb6b47f3a571e9d4a183fc7fd6ac9631a3e3/includes/class-newspack-popups-settings.php#L171) 🤦 
We should update it - but I wonder if legacy option name should be handled at all, since this option is meant to be turned on temporarily and sporadically?